metamorph/src/main/java/org/metafacture/metamorph/Metamorph.java

```diff
@@ -48,6 +49,8 @@
 import org.metafacture.metamorph.api.NamedValueReceiver;
 import org.metafacture.metamorph.api.NamedValueSource;
 import org.metafacture.metamorph.api.SourceLocation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
```
error: package org.slf4j does not exist
```

```diff
diff --git metamorph/build.gradle metamorph/build.gradle
index 5ed1bcee..a14f783d 100644
--- metamorph/build.gradle
+++ metamorph/build.gradle
@@ -24,8 +24,10 @@ dependencies {
   implementation project(':metafacture-flowcontrol')
   implementation project(':metafacture-mangling')
   implementation project(':metafacture-javaintegration')
+  implementation 'org.slf4j:slf4j-api:1.7.21'
   testImplementation 'junit:junit:4.12'
   testImplementation 'org.mockito:mockito-core:2.5.5'
+  testRuntimeOnly 'org.slf4j:slf4j-simple:1.7.21'
 }
```

Reply:
Yeah, you are right. Why do you not commit your fix?

Reply:
I've committed the fix. In general, I don't like to interfere. So just let me know when you're ready to accept commits.